### DISCUSSION TITLE
RevitLookup Ideas

### DISCUSSION THREAD

**Discussion by Nice3point** [jeremytammik/RevitLookup#146] [Ideas]
Share your ideas for future product features with the Revit community. Don't hesitate to express your thoughts, as your input and feedback can help RevitLookup continue to evolve and improve, making it an even more powerful tool for Revit users and developers alike

**Comment by jeremytammik** [comment]
Shared a pointer to this discussion on the blog: [Request for RevitLookup Ideas](https://thebuildingcoder.typepad.com/blog/2023/03/lookup-ideas-jigs-and-acc-docs-access.html#2).

**Comment by jmcouffin** [comment]
**Dockable panel Revit Lookup**
I know that implies quite a refactoring, But to me, Revit Lookup belongs to the tools I really want to have always opened.

**Reply by Nice3point** [reply]
Good idea, I'll think of the best way to do it. Most likely it will be an option, others will also be able to use the classic version

**Reply by Nice3point** [reply]
Although on the other hand it probably requires a large width of the monitor, so that all the data can be comfortably viewed. And maybe a copy of the window on the second monitor would look better

**Reply by jmcouffin** [reply]
Yeah, lots of people still have 24-27" monitors, But I also know a growing number of enthousisasts like me with 36+" monitors.
The workflow would be slightly different I guess, as you would not have a series of opening windows but just one dockable panel showing the data one after the other (maybe with a back and forth arrow set to go up or down in the hierarchy of elements)

**Comment by MexicanCustard** [comment]
Can we get the Built In Parameters back like the pre-2024 version

**Reply by Nice3point** [reply]
what do you mean

**Reply by MexicanCustard** [reply]
![image](https://github.com/user-attachments/assets/3ba92a18-831c-4cca-8d04-ecdd3d6c744a)
